app/lib/actions/katello/organization/manifest_import.rb

```diff
@@ -11,19 +11,25 @@ def plan(organization, path, force)
           manifest_update = organization.products.redhat.any?
 
           sequence do
-            plan_action(Candlepin::Owner::Import,
+            cp_import_task = plan_action(Candlepin::Owner::Import,
                         :label => organization.label,
                         :path => path,
                         :force => force)
-            plan_action(Candlepin::Owner::ImportProducts, :organization_id => organization.id)
+            concurrence do
```

Comment:
Can these really be concurrent?

Reply:
I was only thinking of the Dynflow logic but I think you're right, I didn't consider the products may have changed after the import.  I'll change it back.